prow/plugins.yaml

```diff
@@ -73,6 +73,14 @@ approve:
   implicit_self_approve: true
   lgtm_acts_as_approve: true
 
+# Lower bounds; XS is assumed to be zero.
```

Comment:
The comment should specify that the value is number of lines (not explicitly clear).